modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendar.testcase

```diff
@@ -209,13 +293,49 @@ definition {
 
 	}
 
+	@description = "LRQA-71598 - Verify that the Time Zone selected by default follows the User's preferences."
+	@priority = "3"
+	test DefaultTimeZoneFollowsUserPreferences {
+		Navigator.gotoAccountSettings();
+
+		UserNavigator.gotoPreferences();
+
+		Navigator.gotoNavNested(navNested = "Display Settings");
+
+		PortalSettings.configureTimeZone(timeZone = "(UTC) Coordinated Universal Time");
+
+		Navigator.gotoPage(pageName = "Calendar Page");
+
+		CalendarNavigator.gotoAddMyCalendars();
+
+		Calendar.viewSelectedTimeZone(timeZone = "(UTC) Coordinated Universal Time");
```

Comment:
Here you may assert the "(UTC -02:00) Brasilia Summer Time", cause i think (UTC) Coordinated Universal Time) already is the default used. Thanks